flutter/lib/src/native/java/sentry_native_java.dart

```diff
@@ -40,8 +40,8 @@ class SentryNativeJava extends SentryNativeChannel {
             _startRecorder(
               call.arguments['directory'] as String,
               ScheduledScreenshotRecorderConfig(
-                width: call.arguments['width'] as int,
-                height: call.arguments['height'] as int,
+                targetWidth: call.arguments['targetWidth'] as int,
+                targetHeight: call.arguments['targetHeight'] as int,
```

Comment:
can we avoid the rename? doesn't seem necessary. Also, this being a "configuration", not a status it's implicit that it's a "target" width/height, just like it's a target frame rate at which we want to capture.

Reply:
I renamed it, because it was hard to distinguish between multiple variables which were all called `width` and `height` with some are having the `sourceWidth`, `logicalWidth`, `deviceWidth` and `targetWidth`. So for me it wasn't obvious and a little confusing.
But sure, I can undo the renaming. Can I add some comments, stating that this is the `targetWidth`?